Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM tiangolo/meinheld-gunicorn-flask:python3.7
+
+# Installing netcat for usage in start script
+RUN apt-get update && apt-get install -y netcat
+
+COPY . /app
+WORKDIR /app
+
+RUN pip install -r requirements.txt
```

Comment:
Might be a good idea to first upgrade pip, using `pip install --upgrade pip` as the one pre-installed might not always be the most up-to-date.

Reply:
Will fix!